nanoFramework.CoreLibrary/CoreLibrary.nfproj

```diff
@@ -107,6 +107,7 @@
     <Compile Include="System\EventHandler.cs" />
     <Compile Include="System\Exception.cs" />
     <Compile Include="System\FlagsAttribute.cs" />
+    <Compile Include="System\FormatException.cs" />
```

Comment:
You need to add this file to the non-reflection project too (as linked file).
This requires regenerating the assembly declaration for it and update nf-interpreter.

Reply:
Ahhh...that's why the MDP didn't generate a changed stub!  Will do!

Reply:
Now added to no-reflection project and new checksums added to mscorlib in nanoCLR.

Reply:
Nice. I've fixed the inclusion so it's linked like all the rest. 

Also added back the System folder.